Describe mergeColumnFieldValues and mergeRowFieldValues properties (#2143)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/ExcelExportPivotGridProps/mergeColumnFieldValues.md

---
---
##### shortDescription
Specifies whether to merge neighbouring cells in the column field if they have the same values.

---

> The following image shows the exported pivot grid if you set this property to **true** or leave it unspecified:

Reviewer: ```suggestion
The following images show the exported pivot grid if you set this property to **true** or leave it unspecified
```

Author: Hello, @dirk-pieterse. Tech writers recently had a discussion and we decided not to divide a single sentence with images (or code blocks). It is better to create an unordered list or use some kind of the following text: 
To do A, assign B to the property: (image)
If you need C, set the property to D.

Also I'll write a message in English for you and Armina to sum up all our thoughts on this topic.

Reviewer: Ok

> The following image shows the exported pivot grid if you set this property to **true** or leave it unspecified...

Reviewer 2: ```suggestion
The following image shows the exported pivot grid if you set this property to **true** or leave it unspecified:
```

![DevExtreme PivotGrid: mergeColumnFieldValues set to true](/images/UiWidgets/pivotgrid-merge-column-field-values-true.png)

> ... and in this image this property is set to false:

Reviewer 2: ```suggestion
In the image below, the **mergeColumnFieldValues** property is set to **false**:
```

> In the image below, the **mergeColumnFieldValues** property is set to **false**:

Reviewer: ```suggestion
or set the **mergeColumnFieldValues** property to **false**:
```
